test_pydantic_schemas.py

```
@@ -0,0 +1,409 @@
+    }
+    
+    try:
+        # Test Create schema
+        snapshot_create = ConfigurationSnapshotCreate(**snapshot_data)
+        print(f"   ✅ ConfigurationSnapshotCreate: {snapshot_create.config_type} - {snapshot_create.change_type}")
+        
+        # Test model validation (requires_restart + affected_services)
+        valid_data_copy = snapshot_data.copy()
+        valid_data_copy["requires_restart"] = True
+        valid_data_copy["affected_services"] = ["web"]
+        snapshot_valid = ConfigurationSnapshotCreate(**valid_data_copy)
```

Ruff (0.12.2)

Local variable `snapshot_valid` is assigned to but never used

Remove assignment to unused variable `snapshot_valid`

(F841)